features/in_onderzoek.feature

```diff
@@ -884,34 +1047,77 @@ Functionaliteit: in onderzoek
       Gegeven het systeem heeft een persoon met de volgende gegevens
       | naam                        | waarde    |
       | burgerservicenummer         | 555550001 |
-      En de persoon met burgerservicenummer '555550001' heeft de volgende 'naam' gegevens
-      | naam                          | waarde                  |
-      | voornamen (02.10)             | Piet                    |
-      | geslachtsnaam (02.40)         | Groenen                 |
-      | aanduidingNaamgebruik (61.10) | <aanduidingNaamgebruik> |
-      En de persoon heeft een partner met de volgende gegevens
-      | naam                | waarde    |
-      | burgerservicenummer | 555550002 |
-      En de partner met burgerservicenummer '555550002' heeft volgende 'inOnderzoek' gegevens
-      | naam                          | waarde   |
-      | aanduidingGegevensInOnderzoek | 050200   |
-      | datumIngangOnderzoek          | 20220307 |
+      | geslachtsaanduiding (04.10) | M         |
```

Comment:
Ik begrijp niet waarom dit scenario hier is opgenomen. De kolom 'in onderzoek' in de Voorbeelden tabel wordt ook helemaal niet gebruikt. Ik vermoed dat dit scenario nog niet helemaal gereed is.

Reply:
ik denk dat het erom gaat dat in deze gevallen geen inOnderzoek wordt gezet op aanhef en gebruikInLopendeTekst. Dat zou duidelijker uit de scenariotitel kunnen blijken.

Reply:
titel van scenario (inclusief <omschrijving>) is niet duidelijk. gaat om tonen dat inOnderzoek van partner niet wordt opgenomen wanneer onderzoek gaat over partnergegeven die niet gebruikt wordt